python/pyspark/ml/tests/test_clustering.py

```diff
@@ -461,6 +462,40 @@ def test_distributed_lda(self):
             model2 = DistributedLDAModel.load(d)
             self.assertEqual(str(model), str(model2))
 
+    # TODO(SPARK-51080): Fix save/load for PowerIterationClustering
```

Comment:
`PowerIterationClustering` is not an estimator/transformer/model

can we support save/load of `JavaMLReadable/JavaMLWritable`? @wbo4958

Reply:
Is it possible that we use PowerIterationClusteringWrapper to do the read/write?

Reply:
Then it will save an instance of `PowerIterationClusteringWrapper`, and we may need additionally convert it to `PowerIterationClustering`.
It will be better if we can directly save/load `PowerIterationClustering`